h2o/src/main/scala/org/apache/mahout/h2obindings/H2OEngine.scala

```diff
@@ -26,9 +26,13 @@ import org.apache.mahout.math.drm.logical._
 import org.apache.mahout.h2obindings.ops._
 import org.apache.mahout.h2obindings.drm._
 import org.apache.mahout.h2o.common.{Hadoop1HDFSUtil, HDFSUtil}
+import org.apache.mahout.logging._
 
 /** H2O specific non-DRM operations */
 object H2OEngine extends DistributedEngine {
+
+  private implicit val log = getLog(H2OEngine.getClass)
```

Comment:
yes it should, in fact.  thanks. although it is more about private than final; private values cannot be neighter changed nor overriden anyway. I am a bit surprised Scala allows "private final" in the first place.